static/js/src/libs/instant-page/instantpage.js

```diff
@@ -0,0 +1,202 @@
+/*! instant.page v3.0.0 - (C) 2019 Alexandre Dieulot - https://instant.page/license */
```

Comment:
Did you mean to commit this file? If so, then why does it need to be added to the package.json?

Reply:
Now added to .gitignore